### ISSUE TITLE
Question Regarding the Rewards and Advantage handling in Mathmatics Reasoning

### ISSUE THREAD

**Issue by littlestone111** [NVlabs/GDPO#10]
Hi authors,

Thank you for open-sourcing your code and for your interesting work on MORL! I am trying to replicate the experiment with two reward settings in the paper. However, it seems the shared code is mostly for RLLA. Could you please provide suggestions on where to find the mathematical reasoning training script and reward handling? I am particularly interested in how different-length rewards are handled.

Thank you!
